src/Modal/Modal.scss

```diff
@@ -13,10 +15,45 @@
   }
 }
 
+.modal.is-ie {
+  // fix garbage browser
```

Comment:
I mean... I agree, but maybe we should remove this comment or make it a little less negative. 'fix browser that likes to things its own way' has a nice ring, no?

Reply:
Heh, ok fair. I wrote that when I was a bit frustrated :stuck_out_tongue:

Reply:
It's up to you! Perhaps I am being too cautious. My rationale was that if we're supporting a particular browser and people on our platform use that browser and Paragon is a component library for that platform we might want to remain unbiased in our comments :P. But again, maybe the caution is unnecessary.

Reply:
Yeah you have a point. I like your suggestion. It's snarky in a nice way :laughing: